arcade/experimental/examples/tesellation.py

```diff
@@ -88,19 +87,19 @@ def __init__(self, width, height, title):
             "f",
             # x, y
             [
-                -1.0, 0.0,
-                -0.5, 1.0,
-                0.5, -1.0,
-                1.0, 0.0,
-            ]
+                -1.0,
+                0.0,
+                -0.5,
+                1.0,
+                0.5,
+                -1.0,
+                1.0,
+                0.0,
+            ],
+        )
```

Comment:
A bit awkward considering the comment indicate each line respresents an x and y value.